lib/commands/preload.ts

```diff
@@ -83,9 +88,11 @@ export default class PreloadCmd extends Command {
 	public static usage = 'preload <image>';
 
 	public static flags: flags.Input<FlagsDef> = {
+		// TODO: Replace with application/a in #v13?
 		app: flags.string({
-			description: 'name of the application to preload',
+			description: 'name of org/name slug of the application to preload',
```

Comment:
`name of` → `name or`
`name or slug of the application to preload`
Actually, couldn't this be as follows? -
```
		app: cf.application,
```

Reply:
Yeah, let's standardise